src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -278,6 +278,8 @@ public static Version GetInstalledAsmVersion2(IEnumerable<Version> versions, ref
                 // Fallback mechanism, look inside libg folders if any of them contains ASM dlls.
                 foreach (var v in versions)
                 {
+                    if (v == null) continue;
```

Comment:
when did this happen?

Reply:
About a week ago I was seeing some nulls in here when using the package manager search as normal.

Reply:
that's odd - can you get a stacktrace please? I can't think of a good reason that the package manager search is looking for installations of ASM - that seems like a mistake if so.

Reply:
No luck getting that stacktrace I'm afraid. Should it happen again, I'll send you a screenshot immediately :)